VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss

```diff
@@ -0,0 +1,30 @@
+@import '~govuk-frontend/govuk/all';
+
+.icons {
+    width: 17px;
+    height: 20px;
+}
+.copy-box {
+    display: none;
+    top: 557px;
+    left: 139px;
+  }
+.copyIDbox {
+    font-family: "GDS Transport", Arial, sans-serif;
```

Comment:
Use the mixins instead of setting each style
`@include govuk-font(14, $weight: 'regular', $tabular: false, $line-height: false);`

Reply:
updated to use mixins